srcpkgs/nyxt/template

```diff
@@ -1,19 +1,19 @@
 # Template file for 'nyxt'
 pkgname=nyxt
-version=2.2.4
-revision=3
+version=3.2.0
+revision=1
 create_wrksrc=yes
 build_style=gnu-makefile
 make_build_target=all
-hostmakedepends="sbcl git"
+hostmakedepends="sbcl git pkg-config"
 makedepends="webkit2gtk libfixposix-devel libgirepository"
 depends="dbus xclip enchant2 webkit2gtk libfixposix libgirepository"
 short_desc="Keyboard-oriented, extensible web-browser"
 maintainer="0x0f0f0f <sudo-woodo3@protonmail.com>"
```

Comment:
Could you also please change `maintainer` to `Orphaned <orphan@voidlinux.org>` since they haven't contributed/updated the package since 2019.

Reply:
Of course. Done